.gitignore

```diff
@@ -10,6 +10,9 @@ pids
 *.pid
 *.seed
 
+# API Keys
+src/client/api/google-api-key.js
```

Comment:
can we rename this to config.js or api-keys.js so if/when we want to add new api keys we can just add them to that file. Also if this is modified, can the person modifying send the file with the api keys as a snippet on slack so we can get the update.